ThingIFSDK/ThingIFSDK/Trait/TraitCommandForm.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TraitCommandForm.swift
+//  ThingIFSDK
+//
+//  Created on 2016/12/08.
+//  Copyright (c) 2016 Kii. All rights reserved.
+//
+
+import Foundation
+
+/**
+Form of a command for trait version thing.
+
+This class contains data in order to create `Command` with
```

Comment:
@yongpingchen Request payloads of post new command for non trait and trait are different so I think class representing the payloads should be different.

I guess your point is that `TraitCommandFrom` and `Command` is not symmetry don't you? If so, I feel little strange for the asymmetry too. Application programmer may be confused.

So how about changing return value? I think this strange feeling comes from mismatch of return value type and argument value type.

If return value is String which is return ID, I think the strange feeling disappear.
REST API response of post new command is command ID, so this is more natural.

What do you think?